DMTN-222.tex

```diff
@@ -408,6 +408,89 @@ \section{Curated Calibrations}
 
 This will ensure that the calibrations can be chained into the main collection as detailed above.
 
+\section{Batch Process Submission (BPS) Templates}
+
+We have put together a set of convenient
```

Comment:
If these are github-formatting newlines, ignore this;  I think technotes are supposed to be one-sentence-per-line, so they diff conveniently.